openassessment/xblock/static/js/src/lms/oa_response.js

```diff
@@ -523,11 +508,11 @@ OpenAssessment.ResponseView.prototype = {
     fileUpload: function() {
         var view = this;
         var fileUpload = $(".file__upload");
-        fileUpload.addClass("is--disabled");
+        fileUpload.prop('disabled', true);
 
         var handleError = function(errMsg) {
             view.baseView.toggleActionError('upload', errMsg);
-            fileUpload.removeClass("is--disabled");
+            fileUpload.prop('disabled', false);
```

Comment:
Right, and 

> $( "input" ).prop( "disabled", false );

 will remove the disabled attribute. At least that's what I saw when I tried debugging it, and it seems consistent with their documentation.